tests/Integration/Behaviour/Features/Scenario/Attribute/attribute_management.feature

```diff
@@ -0,0 +1,64 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s attribute
+@restore-all-tables-before-feature
+@attribute-management
+Feature: Attribute group management
+  PrestaShop allows BO users to manage product attribute groups
+  As a BO user
+  I must be able to create, edit and delete product features
+
+  Background:
+    Given shop "shop1" with name "test_shop" exists
+    And language "en" with locale "en-US" exists
+    And language "fr" with locale "fr-FR" exists
+    And language with iso code "en" is the default one
+    And shop group "default_shop_group" with name "Default" exists
+    And I add a shop "shop2" with name "test_shop2" and color "blue" for the group "default_shop_group"
+    And I add a shop "shop3" with name "test_shop3" and color "red" for the group "default_shop_group"
+    And I add a shop "shop4" with name "test_shop4" and color "red" for the group "default_shop_group"
+    And I create attribute group "attributeGroup1" with specified properties:
+      | name        | Color        |
+      | public_name | Public color |
```

Comment:
```suggestion
      | name[fr-FR]        | Couleur        |
      | name[en-US]        | Color        |
      | public_name[fr-FR] | Couleur publique |
      | public_name[en-US] | Public color |
```
The localized values are still not handled like the other behat tests are used to handle this feature